### ISSUE TITLE
[Doc]: Inheritance diagrams

### ISSUE THREAD

**Issue by chahak13** [matplotlib/matplotlib#24424]
### Documentation Link

https://matplotlib.org/devdocs/api/artist_api.html

### Problem

The links to classes in the inheritance diagram are not aligned properly. If you hover over the classes, the tooltip and the class link is for the wrong class. It's off by 1/2. See the figure for reference, when I hover over `PathPatch`, it shows me the link to `FancyArrowPatch`.

![image](https://user-images.githubusercontent.com/29800965/201228440-e30b7584-f7c2-4a99-b982-6e59a4c46416.png)

### Suggested improvement

The tooltips should be aligned properly.

**Comment by QuLogic** [comment]
I've noticed this today as well. I believe it's because the thinner main article causes the image to be scaled down, and either through browser bugginess or the standard being misinterpreted by Sphinx, but the area map does not get scaled down with it.

**Comment by chahak13** [comment]
Yeah, it's probably more with how it's behaving with the browser. I tried it on a 17" screen and a 13" screen, with firefox and edge, the difference in alignment was different in both cases so it isn't a constant problem being introduced but more with how everything is aligned. Though, I'm not sure how to start resolving this or if it is something that we even can easily change on our end.

**Comment by chahak13** [comment]
I just checked. This is not just a problem on this page but, I suspect, the entire docs. I see it on the animation writers page too.

**Comment by astromancer** [comment]
The diagram itself is also missing some classes. In particular, the `Annulus` patch which i added a while ago. See: [here](https://matplotlib.org/devdocs/api/patches_api.html)

**Comment by QuLogic** [comment]
If you mean on the artist page, that list appears manually generated, so it would need to be added explicitly.

**Comment by chahak13** [comment]
Should be resolved by #22273
